hack/verify-govet.sh

```diff
@@ -3,14 +3,10 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 os::golang::verify_go_version
 
-mkdir -p _output/govet
-
-os::build::setup_env
```

Comment:
@smarterclayton apparently this messes up govet in that it hides a lot of errors, from what I read about govet it should run just fine without it, since it analyzes the source code.